### ISSUE TITLE
black screen mp4 playback after each successful playback

### ISSUE THREAD

**Issue by barboboy** [milan-fabian/slideshow-support#25]
Hello,

I've installed the arm v7a 3.18.0 version on an old android 4.4.4 IPTV box.  I am having two issues:

1. Slideshow will not recognize an NTFS formatted USB drive, it only detects a FAT32 USB drive.
2. Some mp4 would just show as a black screen.  I converted a problematic mp4 to android video.  It would then play but after every successful playback, it plays as a black screen when slideshow selects it again to play (random selection of files inside slideshow folder), and it seems as though the mp4 alternates between successful playback and black screen playback when slideshow selects it.  I have not tested any other video format yet.

I would appreciate if you could advise on how to fix above two issues.
Thank you for your time.

Gilbert

**Comment by milan-fabian** [comment]
@barboboy  Hi Gilbert,

1. Can you please verify whether you are seeing the files on NTFS drive in any file manager available on your Android device? Many Android devices doesn't have support for NTFS file system at all, so Slideshow can't read the files.

2. Can you please open Slideshow web interface - menu Settings - Device settings and toggle "Enable enhanced video player" and "Preload items in playlist" (restart Slideshow app after turning it off or on)?

I am looking forward to your feedback.

Milan

**Comment by barboboy** [comment]
Hello Milan,

1. I can confirm that the android 4.4.4 box can see the files on the NTFS USB drive using two different file managers installed on it.

2. I turned off "Enable enhanced video player" and also turned off "Preload items in playlist".  This time the mp4 video does play but another issue has appeared.  Slideshow now exits by itself after a random amount of time (shorter than longer).   The other files I have on the slideshow/slideshow folder on the box are two pictures and one pdf file with two pages.  When there is only the same mp4 video on the slideshow/slideshow folder, it does seem to play indefinitely.  It is when we add those two .JPGs and one .PDF that something goes wrong.  Also, when I remove the mp4 from the slideshow/slideshow folder, it does not crash and the two pictures and pdf are shown correctly.

Is it the android 4.4.4 box the issue?  Or the slideshow apk version on android 4.4.4?  Would updating the android box fix this?  One thing to note, when i use a file manager to open that same mp4, it does not open.  I am not sure if the android 4.4.4 box requires a specific app to do so.  Slideshow app is able to play same mp4 but with above issue as described.

Your feedback would be greatly appreciated.

Gilbert

**Comment by milan-fabian** [comment]
Hi Gilbert, 

Could you please send us a bug report (via the web interface, menu Information - Bug report) for both problems? One approximatelly 30 seconds after you plug in the NTFS drive in the USB port, the other one immediatelly after Slideshow app crashes and you restart it. The bug report will contain logs, which will help us pin-point the cause of these problems. 

Most features of Slideshow is supported by Android all the way back to version 4.0.3. However the hardware decoding varies with each device and not all of them are very good, some needs various quirks to work right (the bug report will help us find out whether this is the case of your device). 

If the manufacturer of you device offers free software upgrade to newer Androis version, I suggest upgrading it. In addition to more features available, the upgrade often fixes various bugs in Android. 

Milan

**Comment by barboboy** [comment]
Hello Milan,

I've got some good news finally.  I ditched that old android 4.4.4 box (don't even know what Chinese brand it was) and tested a mibox 3c and slideshow works perfectly without any tweaking.  The only thing that i could not do on that mibox 3c is allow slideshow root access but ports 8080 and 8021 are available to update items (instead of using usb drives).  These mibox 3c (from Chinese company Xiaomi) are from 2017.  I needed another one so i went for the cheap mibox 4c and i am confident your excellent slideshow apk will be just as fine on it.

Kudos to you for such an excellent digital signage android app.

Gilbert

**Comment by barboboy** [comment]
Hello Milan,

I've got some good news finally.  I ditched that old android 4.4.4 box (don't even know what Chinese brand it was) and tested a mibox 3c and slideshow works perfectly without any tweaking.  The only thing that i could not do on that mibox 3c is allow slideshow root access but ports 8080 and 8021 are available to update items (instead of using usb drives).  These mibox 3c (from Chinese company Xiaomi) are from 2017.  I needed another one so i went for the cheap mibox 4c and i am confident your excellent slideshow apk will be just as fine on it.

Kudos to you for such an excellent digital signage android app.

Gilbert

**Comment by barboboy** [comment]
Hello Milan,

I've got some good news finally.  I ditched that old android 4.4.4 box (don't even know what Chinese brand it was) and tested a mibox 3c and slideshow works perfectly without any tweaking.  The only thing that i could not do on that mibox 3c is allow slideshow root access but ports 8080 and 8021 are available to update items (instead of using usb drives).  These mibox 3c (from Chinese company Xiaomi) are from 2017.  I needed another one so i went for the cheap mibox 4c and i am confident your excellent slideshow apk will be just as fine on it.

Kudos to you for such an excellent digital signage android app.

Gilbert
